heron/spi/src/java/com/twitter/heron/spi/common/Key.java

```diff
@@ -71,6 +71,8 @@
   PACKING_CLASS            ("heron.class.packing.algorithm",   Type.STRING),
   REPACKING_CLASS          ("heron.class.repacking.algorithm", Type.STRING),
   STATE_MANAGER_CLASS      ("heron.class.state.manager",       Type.STRING),
+  AURORA_CONTROLLER_CLASS  ("heron.class.scheduler.aurora.controller",
```

Comment:
given you have only 2 options, you can consider this is a boolean. You could evolve from it in future if needed, but I seriously doubt that.

Reply:
changed to boolean